webrtc/videoroom/lib/videoroom_web/room_channel.ex

```diff
@@ -33,27 +33,42 @@ defmodule VideoRoomWeb.RoomChannel do
     {:noreply, socket}
   end
 
+  @impl true
   def handle_in("answer", %{"data" => %{"sdp" => sdp}}, socket) do
     socket
     |> send_to_pipeline({:signal, self(), {:sdp_answer, sdp}})
 
     {:noreply, socket}
   end
 
+  @impl true
   def handle_in("candidate", %{"data" => %{"candidate" => candidate}}, socket) do
     socket
     |> send_to_pipeline({:signal, self(), {:candidate, candidate}})
 
     {:noreply, socket}
   end
 
+  @impl true
```

Comment:
Isn't putting `@impl true` once enough?

Reply:
And I believe that elixir at its core uses it once: https://github.com/elixir-lang/elixir/blob/e523eaa53d0e16c65b5c330a3963b36721ab037f/lib/elixir/lib/string_io.ex#L191-L202

Reply:
Using `@impl` in each clause makes it catch wrong arity/typo mistakes in each clause, that's why we tend to do so